extra/2-piping.js

```diff
@@ -16,30 +16,31 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+return a + b;
 }
 
-function multiply() {
-
+function multiply( a, b) {
+return a * b;
 }
 
-function format() {
+function format(number) {
+  return `£${number}`;
 
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+const badCode = format((startingValue + 10)*2);
```

Comment:
Nice, but can you do this line using all of your methods rather than maths operands here?